src/node/ext/ldap/base.py

```diff
@@ -1,13 +1,17 @@
 # -*- coding: utf-8 -*-
+from __future__ import print_function
```

Comment:
I dont think this is necessary, `print('foo') works in python 2 OOTB

Reply:
I think you're right. Probably inserted by modernize and missed during cleanup.